Added additional logging documentation for linux services (#770)
observIQ/observiq-otel-collector: docs/installation-linux.md

Logs from the collector will appear in `/opt/observiq-otel-collector/log`. You may run `sudo tail -F /opt/observiq-otel-collector/log/collector.log` to view them.

> Stdout and stderr for the collector process can be via journald. You man run `sudo journalctl -u observiq-otel-collector.service` to view them.

Reviewer: ```suggestion
Stdout and stderr for the collector process are recorded via journald. You man run `sudo journalctl -u observiq-otel-collector.service` to view them.
```
